kale/utils/metadata_utils.py

```diff
@@ -49,21 +49,16 @@ def validate_metadata(notebook_metadata):
                              "the notebook metadata or as an override"
                              .format(required))
 
-    metadata = copy.deepcopy(DEFAULT_METADATA)
+    metadata = DEFAULT_METADATA.copy()
```

Comment:
This really has to be a deepcopy, since `DEFAULT_METADATA` dictionary contains a list.